inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_selector_params.h

```diff
@@ -469,6 +469,8 @@ struct FusedOpsConfiguration {
     bool allow_for_partial_preload;
     // Load index for shuffle fused op
     std::string shuffle_var_name;
+    // Keep original output layout before fusing conv+reorder
```

Comment:
keep sounds like it sholud be kept.. :)  How about Record original output layout before reorder is fused?

Reply:
Ok, I will